src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/CustomDescription.tsx

```diff
@@ -0,0 +1,183 @@
+import { IconText, Text } from '@gnosis.pm/safe-react-components'
+import { makeStyles } from '@material-ui/core/styles'
+import React from 'react'
+import styled from 'styled-components'
+
+import RenderValue from './RenderValues'
+import { styles } from './styles'
+
+import Block from 'src/components/layout/Block'
+import {
+  extractMultiSendDecodedData,
+  isMultiSendDetails,
+  MultiSendDetails,
+} from 'src/routes/safe/store/actions/transactions/utils/multiSendDecodedDetails'
+import Bold from 'src/components/layout/Bold'
+import OwnerAddressTableCell from 'src/routes/safe/components/Settings/ManageOwners/OwnerAddressTableCell'
+import EtherscanLink from 'src/components/EtherscanLink'
+import { humanReadableValue } from 'src/utils/humanReadableValue'
+import Collapse from 'src/components/Collapse'
+import { useSelector } from 'react-redux'
+import { getNameFromAddressBook } from 'src/logic/addressBook/store/selectors'
+import Paragraph from 'src/components/layout/Paragraph'
+import LinkWithRef from 'src/components/layout/Link'
+import { shortVersionOf } from 'src/logic/wallets/ethAddresses'
+import { Transaction } from 'src/routes/safe/store/models/types/transaction'
+
+export const TRANSACTIONS_DESC_CUSTOM_VALUE_TEST_ID = 'tx-description-custom-value'
+export const TRANSACTIONS_DESC_CUSTOM_DATA_TEST_ID = 'tx-description-custom-data'
+
+const useStyles = makeStyles(styles)
+
+const TxDetailsMethodName = styled(Text)`
+  text-indent: 4px;
+`
+const TxDetailsMethodParam = styled.div`
+  text-indent: 8px;
+`
+const InlineText = styled(Text)`
+  display: inline-flex;
+`
+const TxDetailsContent = styled.div`
+  padding: 8px 8px 8px 16px;
+`
+
+const TxInfo = styled.div`
+  padding: 8px 8px 8px 16px;
+`
+
+const MultiSigCustomData = ({ tx, order }: { tx: MultiSendDetails; order: number }): React.ReactElement => {
+  const classes = useStyles()
+  const methodName = tx.data?.method ? ` (${tx.data.method})` : ''
```

Comment:
```suggestion
  const methodName = tx.data?.method?.toString() || ''
```

Reply:
but you're missing the parentheses that surround the method name

Reply:
oh, missed it, nevermind then